src/main/kotlin/org/jitsi/nlj/rtcp/KeyframeRequester.kt

```diff
@@ -114,7 +94,13 @@ class KeyframeRequester : TransformerNode("Keyframe Requester") {
     private var hasPliSupport: Boolean = false
     private var hasFirSupport: Boolean = true
 
+    /**
+     * At least one method is supported, AND we haven't sent a request very recently.
```

Comment:
this is `*returns true when* at least one method is supported, AND we haven't sent a request very recently.`, right?

Reply:
Yes